docs/conf.py

```diff
@@ -9,7 +9,9 @@
 
 # django-specific configuration
 from django.conf import settings
-settings.configure()
+
+# Removed for breaking doc building with django-guardian installed.
+# settings.configure()
```

Comment:
I'm fine with leaving the comment, usually I just comment on those things to make sure it was an intentional change